edalize/templates/yosys/yosys_nosynth.tcl.j2

```diff
@@ -0,0 +1,40 @@
+yosys -import
+{{ plugins }}
+{% if plugins %}
+yosys -import
+{% endif %}
+
+proc read_files {} {
+{{ file_table }}
+}
+
+proc set_defines {} {
+{% if verilog_defines -%}
+set defines {{ verilog_defines }}
+
+foreach d ${defines} {
+  set key [lindex $d 0]
+  set val [lindex $d 1]
+  verilog_defines "-D$key=$val"
+}
+{%- endif %}
+}
```

Comment:
I think this won't work, we need to pass defines directly to Surelog in ``read_systemverilog`` command.

Reply:
I've removed this part from the template, and added a TODO note to the script that we're going to need to handle defines properly at further stages.